ote_sdk/ote_sdk/algo_backends/test_helpers/e2e_test_system.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Will the whole test suite work w/o e2e?

Reply:
Yes, they should.

Reply:
(The e2e wrappers are transparent classes/decorators if e2e is not installed)